src/Essentials/src/Geolocation/Geolocation.android.cs

```diff
@@ -382,24 +400,21 @@ void ILocationListener.OnStatusChanged(string provider, Availability status, Bun
 
 	class ContinuousLocationListener : Java.Lang.Object, ILocationListener
 	{
-		readonly LocationManager manager;
-
 		float desiredAccuracy;
 
 		HashSet<string> activeProviders = new HashSet<string>();
 
-		internal Action<AndroidLocation> LocationHandler { get; set; }
+		internal Action<AndroidLocation>? LocationHandler { get; set; }
 
-		internal Action<GeolocationError> ErrorHandler { get; set; }
+		internal Action<GeolocationError>? ErrorHandler { get; set; }
 
-		internal ContinuousLocationListener(LocationManager manager, float desiredAccuracy, IEnumerable<string> providers)
+		internal ContinuousLocationListener(LocationManager? manager, float desiredAccuracy, IEnumerable<string> providers)
 		{
-			this.manager = manager;
 			this.desiredAccuracy = desiredAccuracy;
 
 			foreach (var provider in providers)
 			{
-				if (manager.IsProviderEnabled(provider))
+				if (manager != null && manager.IsProviderEnabled(provider))
```

Comment:
```suggestion
				if (manager?.IsProviderEnabled(provider) == true)
```
Although for this I have mix feelings because I do tell jr not to do == true in an if, yet is to make sure we do not get null

Reply:
The new expression also results in having the type `bool?`, so the correct change would be:

    if (manager?.IsProviderEnabled(provider) ?? false == true)

I find that more difficult to read and understand and would keep the original version. What do you think?

Reply:
> The new expression also results in having the type `bool?`, so the correct change would be:
> 
> ```
> if (manager?.IsProviderEnabled(provider) ?? false == true)
> ```
> 
> I find that more difficult to read and understand and would keep the original version. What do you think?

In this case the `== true` is redundant isn't it?

Reply:
Yes it's redundant and since it is even less readable it could be removed.

Reply:
the is a matter of taste, I even inclined to leave it with the && due to readability. That is why I mentioned I had mix feelings :)

Reply:
Ok then I leave this as is.